src/uct/ib/base/ib_md.h

```diff
@@ -83,6 +83,7 @@ typedef struct uct_ib_md_ext_config {
     struct {
         int                  prefetch;     /**< Auto-prefetch non-blocking memory
                                                 registrations / allocations */
+        unsigned             mem_types;    /**< Supported mem types for ODP */
```

Comment:
will need to make it `uint64_t` after https://github.com/openucx/ucx/pull/10116 is merged.

Reply:
fixed